### ISSUE TITLE
Suspicious MCH BLER BER readings 

### ISSUE THREAD

**Issue by DominiqueDePaepe** [5G-MAG/rt-mbms-modem#38]
**Describe the bug**
BER
Failed so far to find a single instance where a BLER > 0 would trigger BER also to turn different from zero. Such while we managed to create situations with totally different  BLER levels from less than 1%, across a few percent, upto 10% and more. Hence I would appreciate to confirmation on this KPI to work and so how. if this MCH BER reading different from zero.

BLER
Also it appears as if the  the terminal is announcing way more frequently CRC errors, than we see BLER figure to show different from zero in the UI and the modem-measurements file.

**To Reproduce**
Reduce incoming RF Power  (and witness CINR drop )  till you see CRC errors come in. 
For myself when selected MCS23 such is at CINR of approx. 15 dB 

**Expected behavior**

While understood that under excessive BLER, the device might not longer  (be able to ) calculate the associated BER, such cannot be always the cases. Also we appreciate some clarification on how possible that BLER readings that are supposed to provide the average for the interval can show/stay zero while the terminal shows is announcing CRC errors already. 

**Screenshots**
(attached)

**Desktop (please complete the following information):**
 No dependency on desktop  (same findings over the API)

**Smartphone (please complete the following information):**
 Not applicable

**Additional context**
![image](https://github.com/5G-MAG/rt-mbms-modem/assets/121034113/b6266b27-91df-4424-ba6b-2a36c33c9d71)
![image](https://github.com/5G-MAG/rt-mbms-modem/assets/121034113/1e03aa14-5fbf-4fdd-af16-4362cf51f4c3)
![image](https://github.com/5G-MAG/rt-mbms-modem/assets/121034113/5afd3802-a609-4669-b391-23662174d7f0)
![image](https://github.com/5G-MAG/rt-mbms-modem/assets/121034113/65a1b5e5-c7dd-4b97-8292-2c0dd71bda1c)

**Comment by dsilhavy** [comment]
@jsroldan Do you have an idea here?

**Comment by jsroldan** [comment]
Yes, the calculation of the BLER is not accurate in the rx right now. The rx miss some of the errors because how it stores and updates the variables. I already fix it, and when I finish I'll do a pull request.

**Comment by dsilhavy** [comment]
@jsroldan Do we have a PR for this?

**Comment by dsilhavy** [comment]
CAS Rel.16 + fixes will be included in the pull request that is going to be issued by @kuehnhammer .
